test/cpp/api/functional.cpp

```diff
@@ -371,3 +371,38 @@ TEST_F(FunctionalTest, LogSigmoid) {
   auto y_exp = torch::log(torch::ones_like(x)/(torch::ones_like(x) + torch::exp(torch::neg(x))));
   ASSERT_TRUE(torch::allclose(y, y_exp, 1e-4, 1e-7));
 }
+
+TEST_F(FunctionalTest, Normalize) {
+  const auto expected = torch::tensor(
+    {{{0.0000, 0.1000, 0.2000, 0.3000, 0.4000},
+      {0.1429, 0.1714, 0.2000, 0.2286, 0.2571}}}, torch::requires_grad().dtype(torch::kFloat64));
+  { // Test #1 
+    auto input = torch::arange(10, torch::requires_grad().dtype(torch::kFloat64));
+    input.resize_({1, 2, 5});
+    auto norm = F::normalize(input, NormalizeOptions().p(1).dim(-1));
+    
+    // reduce to scalar to call .backward()
+    torch::Tensor s = norm.sum();
+    s.backward();
+
+    ASSERT_EQ(s.ndimension(), 0);
+    ASSERT_EQ(input.grad().numel(), 10)
```

Comment:
nit: we are missing `;` at the end of this line.

Reply:
Not sure what happened here, I think I fixed the semicolon on my local and forgot the to push the changes before building my local tests, feels a bit silly now that I wasn't checking the build logs properly..